app/pages/admin/user-settings.cjsx

```diff
@@ -96,7 +96,7 @@ UserLimitToggle = React.createClass
     <div>
       <AutoSave resource={@props.editUser}>
         Subject Limit:{' '}
-        <input type="number" name="subject_limit" min="1" ref="subjectLimit" value={@props.editUser.max_subjects} onBlur={@updateLimit} onChange={handleInputChange.bind @props.editUser} />
+        <input type="number" min="1" ref="subjectLimit" defaultValue={@props.editUser.subject_limit} onBlur={@updateLimit} onChange={handleInputChange.bind @props.editUser} />
```

Comment:
This input needs a name.

I recommend using `value` instead of `defaultValue` and removing the `onBlur` handler which conflicts with the AutoSave parent.